homeassistant/components/time/services.yaml

```diff
@@ -0,0 +1,14 @@
+set_value:
+  name: Set Time
+  description: Set the time for a time entity.
+  target:
+    entity:
+      domain: time
+  fields:
+    time:
+      name: Time
+      description: The time to set.
+      required: false
```

Comment:
What happens if the value is not given, or should this rather be required?

Reply:
good catch, thanks!